security-malware-firewall.php

```diff
@@ -2450,3 +2452,32 @@ function spbc_private_list_add( $ip ) {
 	}
 	return false;
 }
+
+/**
+ * Check filtered variable to get the api key.
+ *
+ * @return void
+ */
+function spbc_activation__maybe_get_api_key()
+{
+	global $spbc;
+	if (
+		$spbc->api_key ||
+		( ! is_main_site() && $spbc->network_settings['ms__work_mode'] != 2 )
+	) {
+		return;
+	}
+	/**
+	 * Filters a getting API key flag
+	 *
+	 * @param bool Set true if you want to get key automatically after activation the plugin
+	 */
+	$is_get_api_key = apply_filters('spbc_is_get_api_key', false);
```

Comment:
This filter can be moved to spbc_activation() without creating this function.
The above condition could be moved there too.